bin/hoc-2015-oct-26-mailing/send-uk-poster-test-mails

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+./send-to-mailing-list 10-26-hoc-uk-posters hoc-2015-oct-26-mailing/test-params.csv
```

Comment:
that pesky test-params, always running away and hiding ;) I'll move it into common..